toolkit/tools/internal/safechroot/safechroot.go

```diff
@@ -81,6 +82,11 @@ var defaultChrootEnv = []string{
 	"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
 }
 
+const (
+	doLazyUnmount   = true
```

Comment:
Perhaps name these `unmountTypeNormal` and `unmountTypeLazy`?

Reply:
Sure, that seems reasonable.